Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
```

Comment:
- [ ] Please change the name of this file to tailwind.css as it gives this error.
![image](https://user-images.githubusercontent.com/74744787/216180648-acbb8d8b-c663-47d5-8040-f75578caa32f.png)

![image](https://user-images.githubusercontent.com/74744787/216180570-4da7eaa7-7e2d-4a08-a4da-0f2c9ce2b198.png)

Reply:
Nope, You need to run "rails tailwindcss:install" :joy:

Reply:
@Peter1907 when I run tailwindcss: install it will generate the same file right? also you didn't include this in your readme file ?!
![image](https://user-images.githubusercontent.com/74744787/216186112-654fc878-3d69-405a-a394-51800f18e583.png)

Reply:
Yes, I should have added that
My bad :smile: